### DISCUSSION TITLE
Logs collector

### DISCUSSION THREAD

**Discussion by Ale4224** [henrygd/beszel#326] [Support]
Hi,
is it possible to collect other containers logs in the log section? Or is it dedicated only to the agent/web logs?

**Comment by henrygd** [comment]
Beszel does not collect logs from any other containers.

In the future perhaps we can allow you to define patterns and collect lines from container logs / syslog that match the patterns, but this is not implemented.

**Reply by razz1k** [reply]
is it still not implemented?

**Reply by azlux** [reply]
@razz1k maybe a dedicated tool for that is better than having an heavy  tool. You can check on https://dozzle.dev/ for logs.
